api/get.js

```diff
@@ -126,8 +139,13 @@ Api.prototype = {
         var lookup = {
             search: '?' + query,
             query: query,
-            path: req.url
+            binding: req.url
         };
+
+        // Append the query string to the base url
+        url += lookup.search;
```

Comment:
okay, thanks. Just for my own understanding here is what we discussed on Skype:

what I mean is you don't need to add it in code
[10:29:12] Americo Albuquerque: just add {{search}} to the url in the proxy section